src/quoteUtils.js

```diff
@@ -20,7 +22,8 @@ function getQuoteFromConnector (connector, params) {
       .query({
         source_account: params.sourceAccount,
         destination_account: params.destinationAccount,
-        destination_expiry_duration: 2
+        destination_expiry_duration: params.destinationExpiryDuration,
```

Comment:
Here the expiry is `2`, but the default is `5`, so if the user doesn't change their code, the behavior changes. That's a breaking change right?